src/main/java/StorageParser.java

```diff
@@ -0,0 +1,50 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class StorageParser {
+
+    public static boolean isDone(String symbol) {
+        if (symbol.equals("✓")) {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
I do not think this is a coding violation but I think a way you could try to implement to shorten this code would be to `return symbol.equals(icon)`. This would just be a suggestion to shorten the code and make it simpler for others to read as well.